packages/salesforcedx-vscode-core/src/commands/forceRefreshSObjects.ts

```diff
@@ -108,7 +109,7 @@ export class ForceRefreshSObjectsExecutor extends SfdxCommandletExecutor<{}> {
 
     channelService.streamCommandOutput(execution);
 
-    if (this.showChannelOutput) {
+    if (this.showChannelOutput && !isSFContainerMode()) {
```

Comment:
This seemed like the best place to get the behavior we are looking for while scoping the impact to just this one command for now. I think it might be nice in the future if there are mechanisms in place to turn up or down the noise (output, terminal, toasts, etc) thrown at the user from our extensions. Especially at startup.

Reply:
At startup are you running any particular command like, `SFDX: Create Project`?

Reply:
In the context of testing this out? No, to test it locally I toggle the env `SF_CONTAINER_MODE` in the launch configuration for `Launch Extensions`, then open an SFDX project. I am happy to show you if you want.